src/compiler/crystal/tools/doc/generator.cr

```diff
@@ -241,6 +251,17 @@ class Crystal::Doc::Generator
     end
   end
 
+  # adds extra line break to TODO lines
+  def isolate_flag_lines(string)
+    string.split(/\n/).map do |line|
```

Comment:
Are you saying to remove the regex completely or just to consolidate the regex into a single method?